models/get_topics.js

```diff
@@ -41,4 +42,22 @@ exports.selectAllArticles = () => {
         }
         return rows;
       });
+}
+
+exports.selectCommentsById = (article_id) => {
+
+    const queryStr = format(`SELECT * FROM comments WHERE comments.article_id = %s ORDER BY comments.created_at DESC;`, article_id)
+    
+    return db.query(queryStr).then(({ rows }) => {
+        return rows;
+      });
+}
+
+exports.checkArticleExists = (article_id) => {
+    const queryStr = format(`SELECT * FROM articles WHERE article_id = %s;`, article_id)
+    return db.query(queryStr).then(({ rows }) => {
+        if(rows.length===0){
+            return Promise.reject({ status: 404, msg: "article not found" });
+        }
+      });
```

Comment:
Your file name is misleading, best to rename it something more generic as we have more than just getTopics in this file (same for the models)